simulators/nav/src/components/HotKeys.vue

```diff
@@ -61,11 +81,17 @@ export default class HotKeys extends Vue {
   @Mutation
   private readonly flipSimulatePercep!:(onOff:boolean)=>void;
 
+  /************************************************************************************************
+   * Private Members
+   ************************************************************************************************/
+  /* Whether or not clicking on the field recenters the field. */
+  private prevDrawMode:FieldItemType|null = null;
+
   /************************************************************************************************
    * Local Getters/Setters
    ************************************************************************************************/
   /* Mapping of hotkeys to functions. */
-  get keymap():Record<string, ()=>void> {
+  get keymap():Record<string, (()=>void)|(Record<string, ()=>void>)> {
```

Comment:
I think a wiki page detailing how the sim works would be a great place to enumerate these hotkeys

Reply:
I think that's really valid. I think that would be an independent change. What do you think?

Reply:
#431

Reply:
Personally I like the idea of shipping up-to-date  documentation with new features. I'd like to see #431 resolved with this PR as well

Reply:
This goes against our 1 issue = 1 commit = 1 PR guideline which I would like to keep for a few reasons primarily based around making reviewing, implementing, and deploying changes faster and easier.

Reply:
Documentation is a necessary part of the development process and integral to the continued success of the team as our code base grows in size and complexity. As our documentation catches up to our code I will expect code changes to be paired with documentation changes in a single issue. This is to prevent our documentation from ever being out of date. Since there is nothing written instructing other people on rover how to use this code, I suppose there is no harm in it arriving at a later date.

Reply:
I completely agree. I think I should have done a better job when I first made the hotkey feature about this documentation, but unfortunately I didn't think of it then. I will make this documentation my next priority.